mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -440,6 +444,26 @@ def _get_job_run_id(self, run):
             return None
         return run.data.tags.get(MLFLOW_DATABRICKS_JOB_RUN_ID, None)
 
+    def _get_table_details(self, run):
+        if run is None:
+            return None
+        table_details = []
+        if run.inputs is not None:
+            for dataset in run.inputs.dataset_inputs:
+                dataset_source = mlflow.data.get_source(dataset)
+                if dataset_source._get_source_type() == _DELTA_TABLE:
+                    source_dict = dataset_source._to_dict()
+                    # check if dataset is a uc table and then append
+                    if (
+                        source_dict.get("is_databricks_uc_table")
```

Comment:
Could these be accessed as properties prior to converting to `dict`? So we don't need to hardcode the strings here

Reply:
@smurching we don't currently expose an attribute for this. the approach you described is hard to implement, since the returned object from `mlflow.data.get_source` is a `DatasetSource`, not a `DeltaDatasetSource`. i think this approach is okay.

Reply:
> the approach you described is hard to implement, since the returned object from mlflow.data.get_source is a DatasetSource, not a DeltaDatasetSource. i think this approach is okay.

Not sure it's that hard to implement, we could check the type of the object + access the property? I still prefer that to using the string field name here

Reply:
FWIW, it looks like importing the DeltaDatasetSource here creates a circular import (tried doing the instanceAs test).

Reply:
I can add these as constants into some util and reference them that way if you prefer that.

Reply:
Oh weird, I would prefer to resolve the circular import if it's not too hard - why is `DeltaDatasetSource` importing this file?